Tests/Fixtures/CarthageProject/Project.xcodeproj/project.pbxproj

```diff
@@ -409,8 +409,8 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
-				SWIFT_VERSION = 4.2;
+				SWIFT_COMPILATION_MODE = wholemodule;
```

Comment:
Does this break old Xcode’s usage?

Reply:
This was a change that Xcode 10 made already.